docs/guide/provider-account-setup.md

```diff
@@ -19,30 +19,21 @@ needs. In the interim, ensure that your AWS API Keys are kept in a safe, private
 1. Create or login to your Amazon Web Services Account and go the the Identity & Access Management (IAM) Page
 2. Click on **Users** and then **Create New Users**. Enter `serverless-admin` in the first field and click **Create**
 3. **View and copy the API Key & Secret. You'll need it in the next step**
-4. In the User record in the AWS IAM Dashboard, look for **Managed Policies** on the **Permissions** tab and click
-**Attach Policy**
+4. In the AWS IAM dashboard, click on the user you just created `serverless-admin`. Look for **Managed Policies** on the **Permissions** tab, and click **Attach Policy**
 5. In the next screen, search for and select **AdministratorAccess** then click **Attach**
 
 ### Setting the AWS API Key & Secret
 
 To start using Serverless and access the AWS API you need to set the AWS API Access Key & Secret. 
 
-#### Quick Setup
```

Comment:
I like the other changes, but don't want to remove the Quick Setup. If you could remove this part from the PR this can be merged.